packages/scss/demo/components/callouts.html

```diff
@@ -12,24 +12,38 @@ <h2 class="demo-divider">Callout</h2>
 </code>
 
 	<!-- Mods -->
-	<div class="callout mod-success">
+	<div class="callout palette-primary">
```

Comment:
Do we need to provide a primary / secondary component in the demo?
I fell like it's not an actual use of callouts.